### ISSUE TITLE
trained weights for base

### ISSUE THREAD

**Issue by world2vec** [lxucs/coref-hoi#11]
Good work.
I only see weights for large, cold you also provider weights for base? That will be much easy to handle for debugging.
Thanks.

**Comment by lxucs** [comment]
Yes I think I do have some trained weights for the base version. If I can find them I will give the link in README.

**Comment by priyadarshi9** [comment]
Hi @lxucs, In the evaluation section, Under "Provided trained models:", there are links to only large models. Could you please provide links to base models as well. 

Thanks!

**Comment by jaikishore4195** [comment]
Hi @lxucs , Any update on the pretrained weights for base model? Thanks.
